plaidml/edsl/__init__.py

```diff
@@ -816,6 +816,26 @@ def TensorIndexes(count):
     """Creates multiple ``TensorIndex`` objects based on ``count``."""
     return [TensorIndex() for i in range(count)]
 
+def Constant(dtype_or_shape, buffer, dims=[], name=''):
+    """Creates a tensor with constant values.
+
+    Args:
+        dtype_or_shape (DType | LogicalShape): A data type or a shape can be
+            specified. If a shape is specified, the `dims` parameter is ignored.
+        dims (list, optional): Specifies the dimensions of the ``Constant``.
```

Comment:
Documentation for the `buffer` parameter is missing

Reply:
Good catch, thanks!